server/src/weaverbird/backends/mongo_translator/registry.py

```diff
@@ -0,0 +1,16 @@
+from typing import Callable
+
+from weaverbird.utils.step_registry import REGISTRY, BackendStepRegistry
+
+
+def mongo_registry() -> BackendStepRegistry:
+    return REGISTRY.registry("mongo")
+
+
+def register(*args, **kwargs) -> Callable:
+    called_with_kwargs = bool(kwargs)
+    step_name = kwargs.pop('step_name', None) or args[0].__name__.replace('translate_', '')
```

Comment:
I think it's better to add an enum and use everytime an explicit `@register(step_name=...)` instead of the magic `translate_...`.
We spend way more time reading and refactoring than writing new code